### PR TITLE
Update ktlint configuration block

### PR DESCRIPTION
**Pull request by nuhkoca**
Hi @JLLeitschuh,

This is a little PR :)

Thanks for an amazing lint check library. It works like a charm indeed! In `README` you set `debug` attribute to true inside `ktlint` block. Since it is `val` in `KtlintExtension` class, I believe we should use it as follows. What do you think? I updated the README accordingly. Thanks in advance.

Lastly, some samples might need to be updated if this is approved.

`val debug: Property<Boolean> = objectFactory.property { set(false) }`

```
ktlint {
  debug.set(true)
}
```

### CODE REVIEW

**Comment by Tapchicoma** [comment]
This block in `README.md` is written in groovy style and having there `debug = true` is valid according to https://docs.gradle.org/current/userguide/lazy_configuration.html:
>     // Note that an assignment statement can be used instead of calling Property.set()

@JLLeitschuh do you know if it possible to provide both kotlin and groovy styles in markdown the same way it is done in Gradle documentation?

**Comment by nuhkoca** [comment]
If you approve I can do the necessary in README.

**Comment by JLLeitschuh** [comment]
@Tapchicoma It's not possible to do that 'selector' style in Markdown. It would require more complicated HTML/CSS than we have available in Markdown.

**Comment by JLLeitschuh** [comment]
I don't have any issue offering suggestions for both Kotlin and Groovy separately.

**Comment by Tapchicoma** [comment]
@nuhkoca sorry for not answering for a long time. If you still want this to be merged - could provide both samples for Groovy and Kotlin styles?

**Comment by nuhkoca** [comment]
Hi @Tapchicoma  yes, I want it, it would be good for me. 

could provide both samples for Groovy and Kotlin styles? - Is this for README only? If so I will do so. Thanks.

**Comment by Tapchicoma** [comment]
As I said in https://github.com/JLLeitschuh/ktlint-gradle/pull/305#issuecomment-554736247, current block is written in Groovy. Would be nice to add same configuration block in kotlin dsl style.

If you want to go even further - I like how it is implemented with collapsable blocks in https://github.com/ben-manes/gradle-versions-plugin/blob/master/README.md. I think Groovy style should be closed by default and Kotlin DSL should be opened.

**Comment by nuhkoca** [comment]
@Tapchicoma  sure, well-noted. I will consider and ping you once the PR is ready. Thanks!

**Comment by nuhkoca** [comment]
Hi @Tapchicoma 

I think README is done in accordance with your requests. Kindly request you to check README whether or not it satisfies you. Otherwise, your comments are welcome.

By the way, Travis fails continuously. Can you please check it, as well?

File: README.md [hunk 1]
```diff
@@ -85,7 +88,22 @@ buildscript {
...
+  repositories {
+    maven("https://plugins.gradle.org/m2/")
+  }
+}
+
+plugins {
+  id "org.jlleitschuh.gradle.ktlint" version "<current_version>"
+}
+```
+</details>
```
Comment (Tapchicoma) [inline README.md]: This block should use kotlin lang :slightly_smiling_face: :
```
```kotlin
```

Another thing that this block should show how-to use this plugin via old classpath approach using kotlin script:
```kotlin
buildscript {
  repositories {
    maven("https://plugins.gradle.org/m2/")
  }
  dependencies {
    classpath("org.jlleitschuh.gradle:ktlint-gradle:<current_version>")
  }
}

apply(plugin = "org.jlleitschuh.gradle.ktlint")
```

Comment (nuhkoca) [reply README.md]: Thanks for reviewing @Tapchicoma, I will correct.

Comment (nuhkoca) [reply README.md]: @Tapchicoma can you please tell me am I supposed to entirely replace my block with the one you sent?

**from**
```
buildscript {
  repositories {
    maven("https://plugins.gradle.org/m2/")
  }
}

plugins {
  id("org.jlleitschuh.gradle.ktlint") version "<current_version>"
}
```

to
```
buildscript {
  repositories {
    maven("https://plugins.gradle.org/m2/")
  }
  dependencies {
    classpath("org.jlleitschuh.gradle:ktlint-gradle:<current_version>")
  }
}

apply(plugin = "org.jlleitschuh.gradle.ktlint")
```

Comment (Tapchicoma) [reply README.md]: @nuhkoca yes, please fully replace your block with what I've proposed

**Comment by nuhkoca** [comment]
Thanks for your interest! Wish a healthy and happy new year!

**Pull request merged**